examples/baselines/diffusion_policy/diffusion_policy/make_env.py

```diff
@@ -1,12 +1,73 @@
+from collections import deque
 from typing import Optional
+
 import gymnasium as gym
 import mani_skill.envs
+import numpy as np
+from gymnasium.spaces import Box
+from gymnasium.wrappers.frame_stack import FrameStack as GymFrameStack
+from gymnasium.wrappers.frame_stack import LazyFrames
 from mani_skill.utils import gym_utils
+from mani_skill.utils.wrappers import CPUGymWrapper, FrameStack, RecordEpisode
 from mani_skill.vector.wrappers.gymnasium import ManiSkillVectorEnv
-from mani_skill.utils.wrappers import RecordEpisode, FrameStack, CPUGymWrapper
 
 
-def make_eval_envs(env_id, num_envs: int, sim_backend: str, env_kwargs: dict, other_kwargs: dict, video_dir: Optional[str] = None, wrappers: list[gym.Wrapper] = []):
+class DictFrameStack(GymFrameStack):
```

Comment:
can this not just inherit the maniskill frame stack wrapper?

I purposely didn't use the original framestack wrapper from gymnasium since it was not properly GPU parallelized.